### DISCUSSION TITLE
Anyway to change set up or spacing for pop-up?

### DISCUSSION THREAD

**Discussion by ghost** [albertkun/21S-ASIAAM-191A#139] [Help Desk] [answered]
Hi all,

I created a simple excel and went through this lab. I like the idea of having these 3 fields in the pop-up but don't necessarily like the spacing between each line.  I'm wondering if anyone knows of a way to change this code to reflect smaller spacing, if not, no worries!

Thanks!
<img width="584" alt="Screen Shot 2021-05-01 at 3 13 46 PM" src="https://user-images.githubusercontent.com/81720023/116799218-97a5e900-aab4-11eb-9ce0-2a1eeb0f35b1.png">
<img width="849" alt="Screen Shot 2021-05-01 at 7 38 32 PM" src="https://user-images.githubusercontent.com/81720023/116799250-e3f12900-aab4-11eb-8b2a-6ff4329a2a81.png">

**Comment by albertkun** [comment, marked as answer]
@ktgreenler Oh no!! The `<br>` is probably responsible for the big breaks. But additionally, the `<p>` or the `<h3>` could be causing it too. Each of these have default "spacing" which would need to be overwritten or tweaked with CSS. Additionally, it could also be extra spaces in the data itself, but it really seems like more of a problem with the elements I mentioned above!

**Reply by ghost** [reply]
Yes, it does seem like it is an issue with those elements. I was actually able to take out the `<br>` and replace them with `''`. This only reduced the space slightly but it definitely helps and looks better!

**Reply by albertkun** [reply]
@ktgreenler that's great!
